datadog_checks_base/datadog_checks/checks/prometheus/mixins.py

```diff
@@ -255,10 +259,10 @@ def get_metric_value_by_labels(messages, _metric, _m, metric_suffix):
         :return: value of the metric_name matched by the labels
         """
         metric_name = '{}_{}'.format(_m, metric_suffix)
-        expected_labels = set([(k, v) for k, v in _metric["labels"].iteritems()
+        expected_labels = set([(k, v) for k, v in iteritems(_metric["labels"])
```

Comment:
Should we use `get` instead?

Reply:
Yes, but to reduce the scope and ease review of this PR I chose just focus on version incompatibilities.